docs/hub/_toctree.yml

```diff
@@ -123,6 +123,8 @@
     title: Static HTML Spaces
   - local: spaces-sdks-python
     title: Custom Python Spaces
+  - local: spaces-embedding
+    title: Embedding your Space
```

Comment:
```suggestion
  - local: spaces-embed
    title: Embed your Space
```
i would not call it "embedding" as ML people see this term and think of something else (a n-vector)

Reply:
Simply replace it with Sharing then maybe ?

Reply:
embed is fine IMO

Reply:
it's specifically "embedding" that might be confusing